dev-tools/iota-indexer-monitoring/docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: "3.7"
+
+services:
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml:ro
+      - prometheus_data:/prometheus
+    ports:
+      - "9090:9090"
+    restart: always
+    networks:
+      - indexer-metrics-net
+
+  grafana:
+    image: grafana/grafana:latest
+    container_name: grafana
+    ports:
+      - "3000:3000"
+    environment:
+      - GF_SECURITY_ADMIN_USER=admin
+      - GF_SECURITY_ADMIN_PASSWORD=admin
```

```suggestion
      - GF_SECURITY_ADMIN_USER=${GF_SECURITY_ADMIN_USER:-admin}
      - GF_SECURITY_ADMIN_PASSWORD=${GF_SECURITY_ADMIN_PASSWORD:-admin}
```
I know you have it hardcoded for the first-time use only, so I just leave that here as a hint.

With that way you have the flexibility to overwrite it via environment vars from outside, instead of passing hardcoded values.

---

Good point, added!